### ISSUE TITLE
getOptionChain stopped working

### ISSUE THREAD

**Issue by mif2000** [joshuaulrich/quantmod#408]
### Description

I use vba and Excel as interface to the BERT UI for R-based support of end-user inquiries to Yahoo  finance.  I've been doing this for 1-2 years, but know very little about the "system" side of BERT.  A few days ago getOptionChain stopped working.  

### Expected behavior

Until recently an R call such as: 
Expirations <- getOptionChain("AAPL", NULL)

would provide a list of expiries, from which I could choose one and (after manipulating the date a bit)
optionChain <-  getOptionChain("AAPL", expiry)

to provide an xts chain with calls and puts tables..  Now these  getOptionChain calls return:
Warning: no data for "AAPL expiry, omitting
  (server response: HTTP error 401.)
and/or Error in Exp/86400: non-numeric argument to binary operator, which seems to indicate a Yahoo authorization error.

### Minimal, reproducible example

```r
Expirations <- getOptionChain("AAPL", NULL)
```

### Session Info

```r
R version 3.5.0 (2018-04-23)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] remotes_2.4.2.1   jsonlite_1.6      quantmod_0.4.24   TTR_0.23-4       
[5] xts_0.11-2        zoo_1.8-5         BERTModule_2.0.14

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.1       lattice_0.20-35  codetools_0.2-15 lubridate_1.7.4 
 [5] grid_3.5.0       magrittr_1.5     stringi_1.4.3    curl_3.3        
 [9] tools_3.5.0      stringr_1.4.0    compiler_3.5.0  
```

**Comment by joshuaulrich** [comment]
I fixed this yesterday in #407. Please install the latest development version of quantmod: `remotes::install_github("joshuaulrich/quantmod")`.

**Comment by mif2000** [comment]
Thanks much
________________________________
